Marlin/Marlin_main.cpp

```diff
@@ -4829,6 +4844,66 @@ inline void gcode_M31() {
 
 #endif // SDSUPPORT
 
+#if ENABLED(PISTON_EXTRUDER)
+  inline void gcode_M35() { //turn on dryer fan
```

Comment:
Right, I have planned to start over, especially since Marlin seems to be going through lots of changes like UBL, and there seem to be some problems running out of ram with large numbers of UBL probe points. I'm waiting for the flood of changes to slow down a bit - don't want to add to your  burden.